lr-cli/cortx_setup/commands/storage/controller_config.py

```diff
@@ -0,0 +1,177 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+from pathlib import Path
+from ..command import Command
+from cortx.utils.conf_store import Conf
+from provisioner.commands import PillarSet
+from provisioner.salt import local_minion_id, function_run
+from .commons import Commons
+
+prvsnr_cluster_path = Path(
+    '/opt/seagate/cortx_configs/provisioner_cluster.json'
```

Comment:
Can we have global config which defines Provisioner-Cluster File Path

Reply:
Sure, there's another path in making which will have all the config data consolidated in lr-cli/config.py, once that patch gets landed will add this change in it.